src/modules/fancyzones/FancyZonesLib/FancyZonesLib.vcxproj.filters

```diff
@@ -284,4 +284,7 @@
       <Filter>Generated Files</Filter>
     </ResourceCompile>
   </ItemGroup>
+  <ItemGroup>
+    <Natvis Include="$(MSBuildThisFileDirectory)..\..\natvis\wil.natvis" />
+  </ItemGroup>
```

Comment:
Also, why is it needed?

Reply:
It was some Visual Studio artifact. Cleaned it up.